Clinical-Genomics/mutacc
# Move tests and coverage to the GitHub actions

The tests + Coveralls GitHub Action is now working because there is no token set for coveralls. It should be created and named: **COVERALLS_REPO_TOKEN**. I haven't done it because I don't have the permissions.

### Added
- Seqkit lib in Dockerfile
### Changed
- Moved unit tests and integration tests from Travis to the GitHub actions (fix #80)
### Fixed
- Unit tests and integration tests
- Added missing part of the pipeline (mutacc synthesize) in docker-compose


**How to test**:
1. Locally, run `docker-compose up`. You should see also the outfiles relative to the last 
2. The 2 new GitHub actions should also pass

**Review:**
Thanks for filling in who performed the code review and the test!

This is patch|minor|major **version bump** because ...
